api/src/paths/project/{projectId}/survey/{surveyId}/sample-site/{surveySampleSiteId}/index.ts

```diff
@@ -81,8 +81,39 @@ PUT.apiDoc = {
                 description: {
                   type: 'string'
                 },
-                survey_sample_site: {
-                  ...(GeoJSONFeature as object)
+                survey_sample_sites: {
+                  type: 'array',
+                  items: {
+                    ...(GeoJSONFeature as object)
+                  }
+                },
+                methods: {
+                  type: 'array',
+                  items: {
+                    type: 'object',
```

Comment:
(not just this line) Don't forget to specify which fields at each level are required, if any